docs/_includes/apiRef/apiKeyDescription.md

```diff
@@ -0,0 +1 @@
+The API key specified in the [Adobe IO Console](https://console.adobe.io/) <button id="apiKeyDescriptionButton" onclick="showApiKeyDescription()">(...)</button><span id="apiKeyDescription">for the UMAPI integration used to authorize this session.  See the [Getting Started](https://adobe-apiplatform.github.io/umapi-documentation/en/getstarted.html) documentation for details.  The header name is `X-Api-Key` and this parameter is its value.</span>
```

Comment:
Need to take out the expand/contract button.  In this and all files in the includes/apiRef.